### ISSUE TITLE
my datasets have 6 classes , i find that the network has so poor performance

### ISSUE THREAD

**Issue by XUYUNYUN666** [milesial/Pytorch-UNet#87]
so poor performance

**Comment by milesial** [comment]
On train or on validation ? Have you tried regularization ?

**Comment by XUYUNYUN666** [comment]
> On train or on validation ? Have you tried regularization ?

validation and train are both poor, I do not know the reason, i have not changed the network at all

**Comment by milesial** [comment]
I really don't know what to tell you... Are the masks and images correctly loaded ? Are you able to overfit on a single input image ?
